bin/gpm

```diff
@@ -53,6 +54,17 @@ if (destOpt.value) {
     destOpt.value = Path.resolve(destOpt.value.replace("~", Ul.home()));
 }
 
+if (inputOpt.value === ".") {
+    // Load and parse the package.json file
+    var packageJsonPath = Path.join(process.cwd(), "package.json")
+      , manifest = JSON.parse(Fs.readFileSync(packageJsonPath));
+
+    // Merge all dependencies on a single Array
+    var dependencies = Object.keys(manifest.dependencies)
+      , devDependencies = Object.keys(manifest.devDependencies);
+    inputOpt.value = dependencies.concat(devDependencies);
```

Comment:
`is_dev` can be set when the `Gpm` instance is initialized:

```js
var pack = new Gpm("foo", { is_dev: true });
```

That means `Gpm.install` will install dev dependencies as well, recursively (will initialize `Gpm` using `is_dev:true`).

So, would be nice to have something like this:

```js
console.log(pack.getDependencies());
```

This will output: `["foo", "bar", "baz"]` or `["foo", "bar", "baz", "mocha"]`, if `is_dev:true`.

Hope you got it! Ask me if you have something that is not clear. Making this method async is not a problem.

On the other side, I'm not really happy with accepting a list of strings in `Gpm` constructor. It's better to handle that in the bin script and calling `dep.install()` for each of them.

Anyway, I'm sure it will be great! :smile: